Source/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs

```diff
@@ -126,6 +126,11 @@ public IFakeOptionsBuilder<T> Implements(Type interfaceType)
                 return this;
             }
 
+            public IFakeOptionsBuilder<T> Implements<TInterface>()
+            {
+                return Implements(typeof (TInterface));

```

Comment:
1. nice typeparam name
2. this is optional, and I wouldn't bother commenting if I weren't already going to ask for changes, but the FakeItEasy convention is to start in-class method calls with `this.`. There's a warning omitted by the build, and the owners often take care of these kinds of changes, but you're keen, so I figure I'd push my luck and see how you felt about making the change anyhow.